solidity/contracts/helpers/BadERC20Token.sol

```diff
@@ -0,0 +1,101 @@
+pragma solidity ^0.4.24;
+import './interfaces/IBadERC20Token.sol';
```

Comment:
Bad or NonStandard?

Reply:
"bad" for tests, "nonstandard interface" for production, silly naming